radlab-ui/webapp/src/routes/Deployments.tsx

```diff
@@ -112,7 +163,9 @@ const Deployments: React.FC<DeploymentsProps> = () => {
 
   return (
     <RouteContainer>
-      <div className="flex mb-6">
+      <Filter filters={["status", "module"]} clearFilter={clearFilter} />
```

Comment:
Can we have the Filters below the tabs?
<img width="1158" alt="image" src="https://github.com/GoogleCloudPlatform/rad-lab/assets/69013624/98439374-ddfb-4341-b8d0-4fa15fba17b3">